haskell-src/Concordium/Types.hs

```diff
@@ -1148,3 +1155,21 @@ createAlias (AccountAddress addr) count = AccountAddress ((addr .&. mask) .|. re
 -- Template haskell derivations. At the end to get around staging restrictions.
 $(deriveJSON defaultOptions{sumEncoding = TaggedObject{tagFieldName = "type", contentsFieldName = "address"}} ''Address)
 makeLenses ''CommissionRates
+
+-- | The index into the global token table.
+newtype TokenIndex = TokenIndex {theTokenIndex :: Word64}
+
+-- | A token amount. The encoding is  VLQ/ULEB128 (TODO: (drsk) tbd). To compute the
```

Comment:
We need to decide which encoding we will be using.

Reply:
I have an implementation in this PR: https://github.com/Concordium/concordium-node/pull/1359/files#diff-1a728bc8e6dd831cf39af11900a8cc0d235e3df23716e929b4d95f270d83e11dR32